exec/java-exec/src/test/java/org/apache/drill/exec/store/parquet/TestParquetMetadataCache.java

```diff
@@ -398,6 +398,23 @@ public void testDrill4877() throws Exception {
 
   }
 
+  @Test // DRILL-3867
+  public void testMoveCache() throws Exception {
+    String tableName = "nation_move";
+    String newTableName = "nation_moved";
+    test("use dfs_test.tmp");
+    test("create table `%s/t1` as select * from cp.`tpch/nation.parquet`", tableName);
+    test("create table `%s/t2` as select * from cp.`tpch/nation.parquet`", tableName);
+    test(String.format("refresh table metadata %s", tableName));
+    checkForMetadataFile(tableName);
+    File srcFile = new File(getDfsTestTmpSchemaLocation(), tableName);
+    File dstFile = new File(getDfsTestTmpSchemaLocation(), newTableName);
+    FileUtils.moveDirectory(srcFile, dstFile);
+    Assert.assertFalse("Cache file was not moved successfully", srcFile.exists());
+    int rowCount = testSql(String.format("select * from %s", newTableName));
+    Assert.assertEquals(50, rowCount);
+  }
+
```

Comment:
The tests here don't verify opening a version 1.10 metadata file with this 1.11 change. What happens? Can we read old files?